doc/manual/installation.md

```diff
@@ -300,7 +300,7 @@ You should receive `syntax is okay` and `test is successful` messages. If you re
 Visit YOUR_SERVER in your web browser for your first Huginn login. The setup has created a default admin account for you. You can use it to log in:
 
     admin
-    password
+    yourpassword
```

Comment:
I see how this can be confusing, the [default password is actually password](https://github.com/cantino/huginn/blob/master/db/seeds.rb#L7). Maybe we should change it to `password` in the `SEED_PASSWORD` example?